frontend/src/components/roadmap/sections/RoadmapProcess.tsx

```diff
@@ -39,14 +39,15 @@ export default function RoadmapProcess() {
 
   return (
     <RoadmapPageSection className={"bg-white"} title={t("title")}>
-      <p className="margin-0 tablet-lg:margin-bottom-2 line-height-sans-4 tablet:font-sans-md">
-        {t("sectionSummary")}
-      </p>
+      <p>{t("sectionSummary")}</p>
       {roadmapProcesSectionGridRows.map((sectionRow, sectionRoadIdx) => (
-        <div className="grid-row" key={`roadmap-process-row-${sectionRoadIdx}`}>
+        <div
+          className="grid-row grid-gap"
+          key={`roadmap-process-row-${sectionRoadIdx}`}
+        >
           {sectionRow.map((sectionRowItem) => (
             <div
-              className="grid-col-12 tablet-lg:grid-col-6 tablet-lg:padding-right-5"
+              className="tablet:grid-col-6"
               key={`roadmap-process-${sectionRowItem.title}`}
             >
```

Comment:
- Use grid gap instead of padding on columns

(Note: This could probably use the `Grid` component instead of manually adding grid classes)